ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiCommandCompletionService.scala

```diff
@@ -49,9 +49,13 @@ private[apiserver] final class ApiCommandCompletionService private (
 
         completionsService
           .getCompletions(offset, request.applicationId, request.parties)
+          .via(logger.debugStream(completionsLoggable))
           .via(logger.logErrorsOnStream)
     }
 
+  private def completionsLoggable(response: CompletionStreamResponse): String =
+    s"Responding with completions: ${response.completions.map(_.toProtoString).mkString("[", ",", "]")}"
```

Comment:
We need to dial down the verbosity of this a bit. All we really need is a command id and status. Running it through proto introduces  yet another naming flavour i.e. `command_id`

Reply:
Fixed 👍